assets/js/data/cart/reducers.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Internal dependencies
+ */
+import { ACTION_TYPES as types } from './action-types';
+
+/**
+ * Reducer for receiving items related to the cart.
+ *
+ * @param   {Object}  state   The current state in the store.
+ * @param   {Object}  action  Action object.
+ *
+ * @return  {Object}          New or existing state.
+ */
+const reducer = ( state = { errors: [] }, action ) => {
```

Comment:
I wonder if we should split up things a bit more for the cart state (which will allow for potentially splitting up reducers as this store grows - which it will).  Something like:

```jsx
const cartStoreState = {
    cartEntity: null,
    errors: [],
    meta: {
        applyingCoupon: '',
        removingCoupon: '',
    }
};
```

That way down the road we could split those different state slices into their own reducer and combine them using `combineReducers`.  It'll make it easier for testing and maintenance.